.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "LineOfCode"]
+	path = LineOfCode
+	url = git@github.com:gapisback/LineOfCode.git
```

Comment:
@gregthelaw -- Please review this carefully. I have not worked with submodules myself before, so am not entirely sure if this specification, alone by itself, is sufficient.

Reply:
I haven't used submodules either so am not in a position to review unfortunately. Seems to work though! :)